src/test/org/wojcieszko/blockhaus/BlockHausTest.java

```diff
@@ -0,0 +1,55 @@
+package test.org.wojcieszko.blockhaus;
+
+import org.junit.Test;
+import org.junit.Before;
+import org.junit.After;
+import org.wojcieszko.blockhaus.BlockHaus;
+
+/**
+ * BlockHaus Tester.
+ *
+ * @author <Authors name>
+ * @version 1.0
+ * @since <pre>cze 1, 2018</pre>
+ */
+public class BlockHausTest {
+
+    @Before
+    public void before() throws Exception {
+    }
+
+    @After
+    public void after() throws Exception {
+    }
+
+    /**
+     * Method: getBlockHeight()
+     */
+    @Test
+    public void testGetBlockHeight() throws Exception {
+
+        BlockHaus blockHaus = new BlockHaus();
+        int[] height = {1, 9, 5, 3, 4};
+        blockHaus.setBlockHeight(height);
+        blockHaus.viewAnalysis();
```

Comment:
Dla tych danych to bym się spodziewał wyniku
0 6 5 0 1
A dostaję:
0, 6, 4, 0, 4

Reply:
Nawet wiem czemu... sam się oszukałem :face_with_head_bandage: